### PR TITLE
EVG-17262: Show timezone

### PR DESCRIPTION
**Pull request by sophstad**
EVG-17262

### Description

- Show timezone when reporting timestamps. I opted to report it in `GMT+N` format, which I feel is understandable but not too verbose. All the options are enumerated [here](https://date-fns.org/v2.29.3/docs/format).
- Omit timezone in a few particularly space-constrained places

### Screenshots

Version page:
image

Hosts table:
image

### Testing

- Updated unit tests

### CODE REVIEW

**Review by khelif96 — CHANGES_REQUESTED** [review]
I don't think GMT + some offset is the best way to surface the timezone. It doesn't help me understand what time zone I'm looking at because  I don't have the timezone to offset mappings memorized in the same way seeing the 3-letter code for `est`. Users also set their timezone in the readable format in the settings so showing the GMT + Offset everywhere else is confusing.

File: src/utils/string/index.ts [hunk 1]
```diff
@@ -113,16 +114,20 @@ export const getDateCopy = (
...
+  const { tz, dateOnly, omitSeconds, omitTimezone } = options || {};
   let { dateFormat } = options || {};
   if (!dateFormat) {
     dateFormat = "MMM d, yyyy";
   }
   const finalDateFormat = dateOnly
     ? dateFormat
-    : `${dateFormat}, h:mm${omitSeconds ? "" : ":ss"} aa`;
+    : `${dateFormat}, h:mm${omitSeconds ? "" : ":ss"} aa${
+        omitTimezone ? "" : " O"
```
Comment (khelif96) [inline src/utils/string/index.ts:125]:
```suggestion
        omitTimezone ? "" : " z"
```

This formats the timezones in the readable `EST`/ `UST` format.

**Review by khelif96 — CHANGES_REQUESTED** [review]
> > I don't think GMT + some offset is the best way to surface the timezone. It doesn't help me understand what time zone I'm looking at because I don't have the timezone to offset mappings memorized in the same way seeing the 3-letter code for `est`. Users also set their timezone in the readable format in the settings so showing the GMT + Offset everywhere else is confusing.
> 
> @khelif96 Agree but the timezone name is not an option offered by `date-fns` 😢 you can see all options at the link in the description. We could create some kind of mapping but even that is inconsistent because not all time zones have alphabetical representations (some are only numbered according to GMT). So I suppose we could special-case a few common time zones to represent with abbreviations, but I tend to prefer standardization. Let me know what you think!

It actually is! https://date-fns.org/v2.0.0-alpha.27/docs/Time-Zones#synopsis

From the above example, you can see that using the character `z` in the formatting string will show a timezone in a readable format.
Passing tests after using `z` instead of `O`
image

File: src/utils/string/index.ts (hunk 1 above)
Comment (sophstad) [reply src/utils/string/index.ts:125]: Wow seems like they need to update their docs! That's not what I expected from this...
image

Comment (khelif96) [reply src/utils/string/index.ts:125]: This function is using the format method from `date-fns-tz` not `date-fns` which includes support for readable timezones. https://github.com/marnusw/date-fns-tz#format

**Review by khelif96 — APPROVED** [review]

**Pull request merged**